docs/user/openstack/README.md

```diff
@@ -144,17 +146,19 @@ If you would like to set up an isolated development environment, you may use a b
 
 ## Running The Installer
 
+### Known Issues
+
+We are shipping the 4.2 release with a set of known issues. It is highly recommended that before running the installer, you check out the [known issues doccumentation](known-issues.md). We are trying to figure out workarounds for the time being, until we are able to resolve these bugs in the upcoming releases. To see the latest status of any bug, read through bugzilla or github link provided in that bug's description. If you know of a possible workaround that hasn't been documented yet, please comment in that bug's tracking link so we can address it as soon as possible. Also note that any bug listed in these documents is already a top priority issue for the dev team, and will be resolved as soon as possible. If you find more bugs during your runs, please read the section on [issue reporting](#reporting-issues) to learn how to file a bug report.
+
 ### Initial Setup
 
-Download the [latest versions](https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest/) of the OpenShift Client and installer and uncompress the tarballs with the `tar` command:
+You will need a pull secret to run the installer. If you don't have one, you can get one at [try.openshift.com](try.openshift.com). Then, download the [latest version](https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest/) of the OpenShift Client and installer and uncompress the tarballs with the `tar` command:
```

Comment:
Should be:

```markdown
... [try.openshift.com](https://try.openshift.com).
```

Reply:
Also, do we need these docs around how to use `try.openshift.com`?  Hopefully they explain themselves.  Our in-repo docs already explain how to build from source; after that I think we can assume the presence of an installer binary and just explain how to use it for a given platform.  Attempting to carry docs of try.openshift.com is going to be hard to maintain and I don't see the benefit.

Reply:
Agreed, this whole part is already well explained on try.openshift.com and we don't want to duplicate information.